CONTRIBUTORS.md

```diff
@@ -1,10 +1,6 @@
 # Join the Organization
 
-Adicione por favor o {Seu nome de perfil} e {Seu nome de usuário do GitHub} e submeta a Pull Request.
-
-Por exemplo:
-
-- Gentil D. Pinto [gentildpinto]
+- Romário Nzenguele da Silva [RomarioNzenguele]
```

Comment:
Olá @RomarioNzenguele

Acredito que percebeu mal o que está escrito no [README.md](https://github.com/Community-Dev-Angola/Joining-to-Community/blob/main/README.md) e no arquivo [CONTRIBUTORS.md](https://github.com/Community-Dev-Angola/Joining-to-Community/blob/main/CONTRIBUTORS.md)

Não deve ser deletado nenhuma linha neste arquivo, e o seu nome deve ser adicionado no final da secção **Membros da Organização:**
Apenas isso.
Espero que corrija o mais breve possível para que você seja adicionado na comunidade 😃

Reply:
Tomei boa nota!